release-test/pom.xml

```diff
@@ -181,12 +181,22 @@
 				<executions>
 					<execution>
 						<id>downloadRegressionSuite</id>
-						<phase>test</phase>
+						<phase>clean</phase>
```

Comment:
Should not have been changed but we leave it and change it the next time we change the file due to that the jenkins job takes a long time to complete